sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_tables_inventory_v1/schema.yaml

```diff
@@ -30,7 +30,7 @@ fields:
   type: STRING
   description: The table type
 
-- mode: NULLABLE
+- mode: REPEATED
```

Comment:
Is this meant to be an array? The name and the description suggest otherwise @wwyc ?

Reply:
Was the change meant to be to the `owners` field here:
https://github.com/mozilla/bigquery-etl/blob/f95a087e03ff76547bfd122d9baf27fb1700e537/sql/moz-fx-data-shared-prod/monitoring_derived/bigquery_tables_inventory_v1/schema.yaml#L38
?